docs/documentation_routes.js

```diff
@@ -85,6 +91,12 @@ router.get('/templates/check-your-answers', function (req, res) {
   res.redirect('/docs/templates/check-answers')
 })
 
+// Link to download the template
+router.get('/templates/download/:template', function (req, res) {
+  const file = `${__dirname}/views/templates/${req.params.template}`
```

Comment:
it might be worth checking the file exists before trying to issue the download, also I don't know if we need to support Windows or not but it's probably worth using `path.join` just in case.